README.md

```diff
@@ -204,12 +204,10 @@ An error will be raised:
 Components are unit tested directly. The `render_inline` test helper wraps the result in `Nokogiri.HTML`, allowing us to test the component above as:
 
 ```ruby
-require "action_view/component/test_helpers"
-
-class MyComponentTest < Minitest::Test
-  include ActionView::Component::TestHelpers
+require "action_view/component/test_case"
 
-  def test_render_component
+class MyComponentTest < ActionView::Component::TestCase
+  test "render component" do
```

Comment:
Is using strings for test names convention these days? The Rails tests I've looked at recently used method names.

Reply:
Well, that's debatable. If you take a look on the official documentation, you'll see some examples using this syntax. By the way, which convention GitHub adopted internally? 🤔